test/web3c.js

```diff
@@ -39,12 +39,6 @@ describe('Web3', () => {
     });
   }
 
-  it('should retrieve contract keys', async function() {
-    let inst = new web3c(gw);
-    let key = await inst.confidential.getPublicKey('0x62f5dffcb1C45133c670C7786cD94B75D69F09e1');
```

Comment:
For v 1.0, we can no longer use `getPublicKey` without first having deployed a confidential contract. The updated test below reflects that.